src/coreclr/src/inc/bundle.h

```diff
@@ -34,20 +34,21 @@ struct BundleFileLocation
     bool IsValid() const { LIMITED_METHOD_CONTRACT; return Offset != 0; }
 };
 
-typedef bool(__stdcall BundleProbe)(LPCWSTR, INT64*, INT64*);
+typedef bool(__stdcall BundleProbe)(LPCSTR, INT64*, INT64*);
 
 class Bundle
 {
 public:
-    Bundle(LPCWSTR bundlePath, BundleProbe *probe);
-    BundleFileLocation Probe(LPCWSTR path, bool pathIsBundleRelative = false) const;
+    Bundle(LPCSTR bundlePath, BundleProbe *probe);
+    BundleFileLocation Probe(LPCSTR path, bool pathIsBundleRelative = false) const;
 
     const SString &Path() const { LIMITED_METHOD_CONTRACT; return m_path; }
     const SString &BasePath() const { LIMITED_METHOD_CONTRACT; return m_basePath; }
 
     static Bundle* AppBundle; // The BundleInfo for the current app, initialized by coreclr_initialize.
     static bool AppIsBundle() { LIMITED_METHOD_CONTRACT; return AppBundle != nullptr; }
-    static BundleFileLocation ProbeAppBundle(LPCWSTR path, bool pathIsBundleRelative = false);
+    static BundleFileLocation ProbeAppBundle(LPCSTR path, bool pathIsBundleRelative = false);
+    static BundleFileLocation ProbeAppBundle(SString path, bool pathIsBundleRelative = false);
```

Comment:
There is a room for internal refactoring in CoreCLR, to avoid using `LPCWSTR` in some cases. I added this overload to reduce the diff size.

I think we can compile installer with cl.exe's [`/utf-8`](https://docs.microsoft.com/en-us/cpp/build/reference/utf-8-set-source-and-executable-character-sets-to-utf-8?view=msvc-160) option and remove more ToUnicode conversions on Windows.

Reply:
The whole runtime is internally based on 16 bit strings to match the managed char format. I am not sure if converting parts in it to use UTF8 would be beneficial.

Reply:
True, I saw that at some callsites of BundleProbe, we have both W and A variants of the same strings present, and thought that they could benefit with `LPCSTR` overload.